### DISCUSSION TITLE
Awesome Risor!

### DISCUSSION THREAD

**Discussion by rubiojr** [deepnoodle-ai/risor#262] [Show and tell]
At the very least, the language deserves its own awesome-* repo!

I couldn't find one, so I took the liberty to create one at [github.com/rubiojr/awesome-risor](https://github.com/rubiojr/awesome-risor).

If you have links, please send them my way :love_letter:!

**Comment by myzie** [comment]
Nice, I appreciate this @rubiojr!

@applejag @luisdavim @edhemphill @alexaandru @runsisi Hope you don't mind me tagging you here. If you have any Risor scripts or projects that you'd be interested in sharing, this would be a great place to do it.

Some of the larger Risor projects I know of are in proprietary products rather than in open source at this point.

**Reply by alexaandru** [reply]
Certainly don't mind :-) but also don't have anything yet. I am working on a text editor, which will definitely use risor to power it's config/plugins systems, but before I get there, I wanted to have great support for TreeSitter so I've been working quite a bit on https://github.com/alexaandru/go-sitter-forest . Now that my text editor has support for ~400+ languages, I can move onto next milestones, of which Risor is one :-) Will ping here and in awesome-risor when that is completed. Cheers!
